Lib/calendar.py

```diff
@@ -38,8 +42,36 @@ def __str__(self):
 
 
 # Constants for months referenced later
-January = 1
-February = 2
+class Months(IntEnum):
+    __order__ = "JANUARY FEBRUARY MARCH APRIL MAY JUNE JULY AUGUST SEPTEMBER OCTOBER NOVEMEMBER DECEMBER" 
+    JANUARY = 1
+    FEBRUARY = 2
+    MARCH = 3
+    APRIL = 4
+    MAY = 5 
+    JUNE = 6
+    JULY = 7
+    AUGUST = 8
+    SEPTEMBER = 9
+    OCTOBER = 10
+    NOVEMEMBER = 11
+    DECEMBER = 12
+
+(JANUARY, FEBRUARY, MARCH, APRIL, MAY, JUNE, JULY, AUGUST, SEPTEMBER, OCTOBER, NOVEMEMBER, DECEMBER) = [month.value for month in Months]
```

Comment:
use `global_enum` instead.

https://docs.python.org/3/library/enum.html#enum.global_enum

https://discuss.python.org/t/define-constants-for-month-numbers-in-calendar-module/25999/8

Reply:
Hi, do you suggest doing it for Months only or on weekdays as well?
```
▶ ./python                         
Python 3.12.0a7+ (heads/main:d1e4917e06, Apr 19 2023, 17:01:00) [GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from calendar import JANUARY
>>> JANUARY
calendar.JANUARY
>>> JANUARY.value
1
>>> from calendar import MONDAY
>>> MONDAY
calendar.MONDAY
>>> MONDAY.value
0
```
this will break `from calendar import SUNDAY` and all
am I missing something?

Reply:
> Hi, do you suggest doing it for Months only or on weekdays as well?

on weekdays as well

> I don't know if this will break `from calendar import SUNDAY` and all

Yes, it does break code.

But according to the [doc](https://docs.python.org/3/library/calendar.html#calendar.setfirstweekday), IMO people should not rely on its type.

We should do checks around, and then updating with a note should be fine.

Reply:
>IMO people should not rely on its type

okay, I will wait until others acknowledge

Reply:
Yes, use `@global_enum` .  Just as a note, if you didn't use the global_enum decorator your last line would just be:

    MONDAY, TUESDAY, ..., SUNDAY = WeekDays

Make the class names singular, i.e. `Weekday` and `Month`.

It shouldn't break any code, since `Month.SUNDAY == 1`

Reply:
okay